week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,43 +15,68 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
-
+  let newArr = []; //to make an array with numbers
+  for (i = 0; i <= arr.length; i++) {
+    if (typeof arr === "number") {
+      newArr = arr.push[i];
+    }
+    for (j = 0; j <= newArr.length; j++) {

```

Comment:
This will break because newArr.length will return (for example), the number 7

However because arrays start at zero (e.g. arr[0], arr[1], arr[2]) it will break when it gets to 7.

You can fix this by just using 

`i < arr.length;`